rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerLegacyImplTest.java

```diff
@@ -26,6 +26,7 @@
 import co.rsk.peg.*;
 import org.bouncycastle.util.encoders.Hex;
 import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
```

Comment:
Unused

Reply:
Seems that there were some issues with my IntelliJ. I had to restart it and invalidate the cache to make it works right again. However, I'll be more cautious with this next time.